lessons/18-js-introduction/index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+<body>
+    <p>
+        <script>
+        // "Shows a popup to the user with a massage"
+            alert ( 'Hello, World' );
+        </script>
+    </p>
```

Comment:
`<script>` can just be on it's own inside the body, shouldn't be inside a paragraph

```html
<body>
  <p>...</p>
  <script>
    // ...
  </script>
</body>
```